libobs/obs.h

```diff
@@ -2204,6 +2258,13 @@ EXPORT void
 obs_output_set_audio_conversion(obs_output_t *output,
 				const struct audio_convert_info *conversion);
 
+/** Returns the number of video encoders associated with this output */
+EXPORT size_t
+obs_output_get_video_encoder_count(const struct obs_output *output);
```

Comment:
This provides a count, but does not provide any info on the exact indexes of how tracks are routed. For example, if you have a layout like `- + + -`, you have two audio encoders, but the output has no clue where they are. There is `obs_output_get_mixers()` which is a badly named function, but provides a bitmask specifying exactly what mixers are routed.

I need to understand this more personally, because I'm pretty sure that function is used for raw output only, but for some reason there are multiple encoded outputs that are using it. This may actually cause audio bugs if say you route track 0 to idx 1, and idx 1 to track 0.